cmd/crc/cmd/constants.go

```diff
@@ -3,5 +3,5 @@ package cmd
 const (
 	commandName      = "crc"
 	descriptionShort = "Local OpenShift 4.x cluster"
-	descriptionLong  = "Deploy local instance of an OpenShift 4.x cluster for development purpose"
+	descriptionLong  = "CodeReady Containers is a CLI tool that manages local OpenShift 4.x clusters optimized for development purposes"
```

Comment:
First of, we are CLI now, but this will be enhanced with a desktop integration soon-ish, also our purpose is to be a tool for use as "testing and development purposes". and third, we only a provide a single instance of an OpenShift 4.x cluster

`CodeReady Containers is a tool that manages a local OpenShift 4.x cluster optimized for testing and development purposes`

Reply:
@robin-owen @code-ready/crc-devel

Reply:
Agreed. But do we really want to mention a specific version here? It will need to be updated for 5.x, 6.x, etc.

Reply:
yes... we do not do 3.x... and in future of a 5.x we might not do CRC ;-)